implants/golem/src/inter.rs

```diff
@@ -113,13 +108,8 @@ fn interactive(ctx: &Context) -> anyhow::Result<()> {
 }
 
 pub fn interactive_main() -> anyhow::Result<()> {
-    let ctx = Context::new(
-        ContextMode::Run,
-        true,
-        &vec![],
-        true,
-    )?;
+    let ctx = Context::new(ContextMode::Run, true, &[], true)?;
```

Comment:
Don’t love it condescending everything on one line here.
Is there a way we can force it to split blocks like this into multiline?

Reply:
Maybe? Personally I like the one line because nothing in the params in complex to make me want to multi-line it. If we had sub-functions in this one I'd see it

Reply:
Fair enough